doc/_static/custom_styles.css

```diff
@@ -82,3 +82,68 @@ div.fa-user-friends::before {
 .dropdown .card-header::first-letter {
   font-weight: bold;
 }
+
+/*** Styling for quickstart interactive tour ***/
+
+.interactive-tour-container {
+  position: relative;
+  display: inline-block;
+  transition: transform 500ms ease-in-out;
+}
+
+.interactive-tour-container .tour-screenshot {
+  display: none;
+  max-width: 100%;
+  height: auto;
+}
+
+.interactive-tour-container img.tour-screenshot-active {
+  display: block !important;
+}
+
+.interactive-tour-container .tour-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.tour-overlay .highlight-rect {
+  fill: none;
+  opacity: 0;
+  stroke: #ee1c24;
+  stroke-linejoin: round;
+  stroke-width: 5;
+}
+
+div#driver-highlighted-element-stage {
+  opacity: 0;
+}
+
+rect.driver-highlighted-element {
+  opacity: 1 !important;
+}
+
+/*** Fix tour navigation buttons' margins, arrows and alignment ***/
+
+div#driver-popover-item .driver-popover-footer {
+  margin-top: 1em;
+}
+
+div#driver-popover-item .driver-navigation-btns {
+  line-height: 1rem;
+}
+
+div#driver-popover-item .tour-arrow {
+    font-size: 2em;
+    line-height: 0;
+    font-weight: bold;
+    pointer-events: none;
+    vertical-align: middle;
```

Comment:
There was a `padding-bottom` suggestion here just a minute ago. I don't understand why that was removed because I think that's necessary.

This is how things look for me in Firefox:

![Selección_048](https://user-images.githubusercontent.com/365293/90832865-5e559080-e30c-11ea-9bdb-fd478e77226c.png)

Reply:
Same, but they look very different for @juanis2112 on macOS. I discuss this here: https://github.com/spyder-ide/spyder-docs/pull/171#discussion_r474305691

Reply:
> Same, but they look very different for @juanis2112 on macOS/Safari

How things look in Chrome? If things look similar in Chrome and Firefox, we should really add that padding because Safari has a very small percentage of users.